internal/controllers/machine/machine_controller.go

```diff
@@ -260,7 +260,8 @@ func (r *Reconciler) reconcile(ctx context.Context, cluster *clusterv1.Cluster,
 		return ctrl.Result{}, err
 	}
 
-	// If the Machine belongs to a cluster, add an owner reference.
+	// If the machine is a stand alone one, meaning not originated from a MachineDeployment, then set it as directly
```

Comment:
```suggestion
	// If the machine is a standalone one, meaning not originated from a MachineDeployment, then set it as directly
```

And for other instances.